pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -144,6 +145,94 @@ func (r *ReconcileComplianceScan) Reconcile(request reconcile.Request) (reconcil
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileComplianceScan) handleRootCASecret(instance *complianceoperatorv1alpha1.ComplianceScan, logger logr.Logger) error {
+	caSecret := &corev1.Secret{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: RootCAPrefix + instance.Name, Namespace: common.GetComplianceOperatorNamespace()}, caSecret)
+	if err == nil {
+		// Already exists, so return.
+		// TODO: Check for expiration and re-create.
```

Comment:
This depends... how long are we planning to keep the CA around?

We might want to consider deleting it in the DONE stage of the compliancescan. What do you think?